synapse/storage/schema/delta/53/drop_sent_transactions.sql

```diff
@@ -13,4 +13,4 @@
  * limitations under the License.
  */
 
-CREATE INDEX IF NOT EXISTS sent_transaction_txn_id ON sent_transactions(transaction_id);
\ No newline at end of file
+DROP TABLE IF EXISTS sent_transactions;
```

Comment:
It would be useful to have a comment here explaining why this is a useful thing to do and some of the history